spec/system/folder_action_system_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'FolderAction', type: :system, js: true do
+  include SystemHelpers
+  # can select folder and team from side-navbar
+  it 'opens folder and team in side-navbar' do
+    login_as_user(:admin)
+    visit('/')
+
+    within(page.find('#side-bar-team-235930340')) do
+      expect(page).to have_text('team1')
+    end
+
+    find('#side-bar-team-235930340').click
+
+    within(page.first('.list-group-item.list-folder-item.bg-blue-one')) do
```

Comment:
While this works I don't think being that specific here is necessary.
```
within(page.first('.team-card-header')) do
    expect(page).to have_text('folder1')  
end
```
This works just as well and is also more readable. :+1: 

I would suggest you go over all these selection-queries and make them 'less complicated' wherever possible. (Just make sure that we don't break functionality by trying to make things easier. Functionality > Readability)

I tested these statements on my machine by setting a `binding.pry` breakpoint (you might need to add the `pry` gem to your `:test` group in your Gemfile) before said statement and running the test with `HEADLESS=false` and then just inspecting the css classes etc. in the Browser Inspector and then testing my new selection query in the console. :video_game: